pkg/reconcilermanager/controllers/reconciler_container_resources.go

```diff
@@ -46,7 +46,7 @@ func ReconcilerContainerResourceDefaults() map[string]v1beta1.ContainerResources
 		reconcilermanager.HelmSync: {
 			ContainerName: reconcilermanager.HelmSync,
 			CPURequest:    resource.MustParse("50m"),
-			MemoryRequest: resource.MustParse("256Mi"),
```

Comment:
@nan-yu I remember we bumped helm memory several times for Autopilot. Should we lower the number here?

Reply:
We bumped helm cpu request for Autopilot, not the memory: https://github.com/GoogleContainerTools/kpt-config-sync/pull/378.
We could probably lower it from `50m` to `10m`.